lib/web/apiserver.go

```diff
@@ -1073,7 +1070,7 @@ func (h *Handler) bindDefaultEndpoints() {
 
 	// Implements the agent version server.
 	// Channel can contain "/", hence the use of a catch-all parameter
-	h.GET("/webapi/automaticupgrades/channel/*request", h.WithUnauthenticatedHighLimiter(h.automaticUpgrades))
+	h.GET("/webapi/automaticupgrades/channel/*request", h.WithUnauthenticatedHighLimiter(h.automaticUpgrades109))
```

Comment:
I don't really like calling them legacy because some parts of the RFD109 will still be used as a fallback for RFD184, and the RFD109 APIs will soon be able to tell to update based on RFD184 configuration.

So you can have legacy updaters using legacy APIs to update according to the new schedule, and new updaters using the new API to update according to the legacy schedule.

As both systems will share a common update logic I don't see channels as legacy, just as a different API service the same content.

The interoperability logic will be introduced in https://github.com/gravitational/teleport/pull/49380

Reply:
I also asked the question here: https://gravitational.slack.com/archives/C04P78M46F2/p1732304167957549